packages/transport-chrome/src/session-client.ts

```diff
@@ -23,8 +23,35 @@ import {
 import { ChannelLabel, nameConnection } from './channel-names';
 import { isTransportInitChannel, TransportInitChannel } from './message';
 import { PortStreamSink, PortStreamSource } from './stream';
-import { Code, ConnectError } from '@connectrpc/connect';
-import { errorToJson } from '@connectrpc/connect/protocol-connect';
+
+const localErrorJson = (e: unknown, m?: unknown) =>
```

Comment:
Can you change name to `(e: unknown, message?: unknown)`? Or is `m` suppose to be something else?